src/main/resources/defaults/data/SkillParticleGeneration.json

```diff
@@ -555,7 +559,7 @@
 		"amountList": [
 			{
 				"value": 1,
-				"chance": 100
+				"chance": 45
```

Comment:
The zero particle case is handled by the client. Shinobu always generates exactly one particle when the client invokes GENERATE_ELEM_BALL.

Reply:
Ah, ok, got it